CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 4.0.0 (2020-05-22)
+
+### New features
+
+- Updated the pipeline to transpile to the [Angular Package Format](https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/preview). [#15](https://github.com/blackbaud/skyux-angular-tree-component/pull/15)
```

Comment:
Should we have an "Added support for `@angular/core@^9`." entry here linking to https://github.com/blackbaud/skyux-angular-tree-component/pull/15? We have it in the other changelogs I've been looking at today, and that PR seems to be fore Angular 9, so I'm not sure what the difference is.

Reply:
Good catch. Done.

Reply:
If we need this here, I also assume we need it below under `4.0.0-rc.0`. I'll go ahead and add it.